tsuru/config/config_test.go

```diff
@@ -27,11 +31,12 @@ func (s *S) TestBoostrapConfigNoConfig(c *check.C) {
 	conf := bootstrapConfig()
 	c.Assert(conf, check.NotNil)
 	expected := newDefaultConf()
-	expected.lastChanges = conf.lastChanges
 	c.Assert(conf, check.DeepEquals, expected)
 }
 
 func (s *S) TestBoostrapConfigFromFile(c *check.C) {
+	now := nowUTC()
+	nowUTC = func() time.Time { return now }
 	fsystem = &fstest.RecordingFs{}
 	f, _ := fsystem.OpenFile(configPath, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0755)
```

Comment:
```suggestion
	f, err := fsystem.OpenFile(configPath, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0755)
	c.Assert(err, check.IsNil)
```

Reply:
I know this isn't part of the PR but should be enforced, could you please make this change?